src/main/AndroidManifest.xml

```diff
@@ -51,6 +51,20 @@
                 android:value=".activities.MainActivity" />
         </activity>
         <activity
+            android:name=".activities.ShareActivity"
+            android:label="@string/share_activity_title">
+            <intent-filter>
+                <action android:name="android.intent.action.SEND" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:mimeType="*/*" />
+            </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.SEND_MULTIPLE" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:mimeType="*/*" />
```

Comment:
Can we limit the mime type somehow? Right now Syncthing shows up for sharing everywhere (even for plain text), and then just shows the "Nothing to share" toast.